cosmoz-page-router.js

```diff
@@ -402,14 +407,14 @@
 					this._activateImport(route, url, eventDetail, importLink);
 				}
 			}
-		},
+		}
 
 		_hasCustomElement(elementName) {
 			const customElements = window.customElements;
-			return Polymer.telemetry.registrations.some(function (element) {
+			return Polymer.telemetry.registrations.some(element => {
```

Comment:
maybe add an null check/array check on Polymer.telemetry.registrations ?

Reply:
I have added a null check on `Polymer.telemetry` and an array check on `Polymer.telemetry.registrations`. Please re-review and check if the change is as desired.

Reply:
I think `Polymer.telemetry` should be replaced by `window.customElements` as per https://developers.google.com/web/fundamentals/web-components/customelements

Reply:
@megheaiulian I tried replacing it, but not sure if I use it correctly because it does not find the start-page element.

Tested the branch against frontend and it fails finding the correct template.